source-mysql/backfill.go

```diff
@@ -153,15 +170,17 @@ func (db *mysqlDatabase) explainQuery(streamID, query string, args []interface{}
 	}
 	db.explained[streamID] = struct{}{}
 
-	// Ask the database to EXPLAIN the backfill query
-	var explainQuery = "EXPLAIN " + query
-	logrus.WithFields(logrus.Fields{
-		"id":    streamID,
-		"query": explainQuery,
-	}).Info("explain backfill query")
-	explainResult, err := db.conn.Execute(explainQuery, args...)
+	// Ask the database to analyze the backfill query.
+	var explainResult, err = db.conn.Execute("ANALYZE "+query, args...)
 	if err != nil {
-		logrus.WithField("query", explainQuery).Warn("unable to execute query")
+		// ANALYZE is not universally supported. Try again with EXPLAIN.
```

Comment:
I believe the bare `ANALYZE` is a MariaDB-ism and the MySQL equivalent is `EXPLAIN ANALYZE`, so it might be worth trying both of those before resorting to a bare `EXPLAIN`.

Reply:
done